src/parser.c

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.8.2.  */
+/* A Bison parser, made by GNU Bison 3.7.6.  */
 
```

Comment:
Probably why so much diff?

Reply:
Yeah, probably!  I should upgrade...

Reply:
I'll build bison if need be and make sure this is less nasty.